conf/mu.appdata.xml

```diff
@@ -18,14 +18,12 @@
         REPL (either running on the connected CircuitPython or MicroPython
         device or as a Jupyter based iPython session in Python3 mode).</p>
         <p>Mu is written in Python and works on Windows, OSX, Linux and
-        Raspberry Pi. The project's public facing website is
-        https://codewith.mu/. We celebrate the work done by users of mu at
-        https://madewith.mu/.</p>
+        Raspberry Pi.</p>
 	</description>
 	<url type="homepage">http://codewith.mu/</url>
 	<url type="help">http://codewith.mu/tutorials/</url>
 	<url type="bugtracker">https://github.com/mu-editor/mu/issues</url>
 	<project_group>mu-editor</project_group>
-	<_developer_name>Nicholas H.Tollervey</_developer_name>
+	<x-developer_name>Nicholas H.Tollervey</x-developer_name>
```

Comment:
This should just be [developer_name](https://www.freedesktop.org/software/appstream/docs/chap-Metadata.html#sect-Metadata-GenericComponent), expect the _ was an accidental prefix

*Pretends they weren't the one that wrote this....*